torch/nn/attention/_flex_attention.py

```diff
@@ -304,15 +311,13 @@ def _create_block_mask(
 
 def _create_empty_block_mask(query, key, value) -> _BlockMask:
     device = query.device
-    kv_len = key.size()[-2]
-    q_len = query.size()[-2]
     return _BlockMask(
         kv_num_blocks=torch.ones([1, 1, 1], dtype=torch.int32, device=device),
         kv_indices=torch.zeros([1, 1, 1, 1], dtype=torch.int32, device=device),
         q_num_blocks=torch.ones([1, 1, 1], dtype=torch.int32, device=device),
         q_indices=torch.zeros([1, 1, 1, 1], dtype=torch.int32, device=device),
-        KV_BLOCK_SIZE=kv_len,
-        Q_BLOCK_SIZE=q_len,
+        KV_BLOCK_SIZE=0,
+        Q_BLOCK_SIZE=0,
```

Comment:
Change the empty block mask to have KV_BLOCK_SIZE and Q_BLOCK_SIZE equals to 0 instead of seq_len. We want KV_BLOCK_SIZE & Q_BLOCK_SIZE to be statics, but seq_len can be dynamic.